packages/frontend/src/components/Info/Accordion.tsx

```diff
@@ -67,33 +69,67 @@ export const InfoAccordion = () => {
-              You receive a ticket for ${exampleBid * 2.5} ETH.`}
+              heading={`Auction pool: ${auctionWinnersCount} tickets`}
+              rule="Tickets from the auction pool will be distributed to the highest bidding participants. The price paid by a winner in that pool is equal to the amount of their bid. All proceeds will go towards Public Goods."
+              example={`You bid ${exampleBid} ETH and end up in the top ${auctionWinnersCount} of the bidders. You receive a ticket for ${exampleBid} ETH.`}
             />
             <Rule
-              heading={`Raffle pool: ${raffleWinnersCount}`}
-              rule={`From participants who bid below the last bid in the auction pool, ${raffleWinnersCount} will be chosen at random.
-              A winner in that pool will receive a ticket for ${reservePrice} ETH. All funds that they bid over that price will be claimable after the raffle is stettled.`}
-              example={`You bid ${exampleBid} ETH and end up below top ${auctionWinnersCount}. If you are selected in the raffle, you pay ${reservePrice}
-              ETH for the ticket and get ${(exampleBid - reservePrice).toFixed(2)} ETH back.`}
+              heading={`Raffle pool: ${raffleWinnersCount} tickets`}
+              rule={`From participants who bid below the last bid in the auction pool, ${raffleWinnersCount} will be chosen at random. A winner in that pool will receive a ticket for ${reservePrice} ETH. All funds that they bid over that price will be claimable after the raffle is settled.`}
+              example={`You bid ${exampleBid} ETH and end up below the top ${auctionWinnersCount}. If you are selected in the raffle, you pay ${reservePrice} ETH for the ticket and get ${(
+                exampleBid - reservePrice
+              ).toFixed(2)} ETH back.`}
             />
             <Rule
-              heading="Golden Ticket: 1"
-              rule="One lucky bidder from the raffle pool will receive a ticket for Devcon 6 totally for free!
-              The Golden Ticket winner will be able to claim the whole amount of their bid after the raffle is settled."
+              heading="Golden Ticket: 1 ticket"
+              rule="One lucky bidder from the raffle pool will receive a ticket for Devcon 6 totally for free! The Golden Ticket winner will be able to claim the whole amount of their bid after the raffle is settled."
             />
             <Rule
               heading="No luck?"
-              rule="In case you don't win, your bid (-2% fee) will be claimable after the raffle is settled."
-              example={`You bid ${exampleBid} ETH and end up not winning a ticket.
-              You can get ${(exampleBid * 98) / 100} ETH back.`}
+              rule="In case you don't win, your entire bid will be claimable after the raffle is settled."
+              example={`You bid ${exampleBid} ETH and end up not winning a ticket. You can get your ${exampleBid} ETH back, minus a 2% sybil resistance fee.`}
```

Comment:
It says nothing about the 2% in the rule but it does in the example, kinda contradictory so we might wanna change that